src/main/java/org/andersen/service/ApartmentService.java

```diff
@@ -3,19 +3,24 @@
 import org.andersen.model.Apartment;
 
 import java.util.List;
-import java.util.Optional;
 
 public interface ApartmentService {
 
-    List<Apartment> getAllApartments();
-
-    Optional<Apartment> findApartmentById(int id);
-
     List<Apartment> sortByStatus(int pageNumber, int pageSize);
 
     List<Apartment> sortById(int pageNumber, int pageSize);
 
     List<Apartment> sortByPrice(int pageNumber, int pageSize);
 
     List<Apartment> sortByNameOfClient(int pageNumber, int pageSize);
```

Comment:
maybe we can use it in the separate interface, like ApartmentSortService
and here you just new fields?

or, create 1 basic interface, but generic))
same for repo